readme.txt

```diff
@@ -2,7 +2,7 @@
 Tags: email, email marketing, emailmarketing, emfluence, api, marketing automation, widget, email widget, email signup, mailing list, newsletter, form, automation
 Requires at least: 4.0
 Tested up to: 6.0
-Stable tag: 2.13
+Stable tag: 2.13.1
```

Comment:
Increase to 2.14

Reply:
Done. Increased WP version to 6.3.2 as well.